### PR TITLE
Add one extra stride for c

### PR DESCRIPTION
**Pull request by xiaoling-yi**
This PR adds an extra half of innermost strides CSR for C to further remove the data contention.

### CODE REVIEW

File: src/main/scala/gemm/BatchGemmSnaxTop.scala [hunk 1]
```diff
@@ -322,6 +331,17 @@ class BatchGemmSnaxTop(TCDMWritePorts: Int = 8) extends BatchGemm {
...
+  // val dir_name = "BatchGemmSnaxTop_%s_%s_%s_%s".format(
+  //   GemmConstant.meshRow,
+  //   GemmConstant.tileSize,
+  //   GemmConstant.meshCol,
+  //   GemmConstant.dataWidthA
+  // )
+  // emitVerilog(
+  //   new BatchGemmSnaxTop(GemmConstant.TCDMWritePorts),
+  //   Array("--target-dir", "generated/%s".format(dir_name))
+  // )
```
Comment (rgantonio) [inline src/main/scala/gemm/BatchGemmSnaxTop.scala:344]: Comment to take out?

**Review by rgantonio — APPROVED** [review]
What's the half-stride for?

**Comment by rgantonio** [comment]
@xiaoling-yi don't forget to add descriptions for the PR! haha 😄

**Comment by xiaoling-yi** [comment]
> @xiaoling-yi don't forget to add descriptions for the PR! haha 😄

ok! But we may not merge it to main. It is a trick to further improve utilization. We need to discuss with compiler side to see if this feature is needed to be merged.

**Pull request closed without merging**